Scripts/JSONFileToCSV/JSONFileToCSV.py

```diff
@@ -0,0 +1,54 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+import json
+import io
+import csv
+
+
+def json_to_csv(data, delimiter):
+    si = io.BytesIO()
+    cw = csv.writer(si, delimiter=delimiter)
+    keys = list(data[0].keys())
+    cw.writerow(keys)
+    for d in data:
```

Comment:
couldn't you simplify these two `for` loops into one `for` loop using `for key, val in data.items():`?

Reply:
i don't think so. think of `data` as a two-dimentional matrix.
data is a list of object.
each object correspond a line in the final csv file (hence the loop over the data record).
for each line, we collect the relevant fields (by the `keys` variable) into a list to append as a csv row.

Reply:
I see you still simplified it though ✓